Extensions/Video/videoruntimeobject-pixi-renderer.js

```diff
@@ -0,0 +1,238 @@
+/**
+ * The PIXI.js renderer for the VideoRuntimeObject.
+ *
+ * @class VideoRuntimeObjectPixiRenderer
+ * @constructor
+ * @param {gdjs.VideoRuntimeObject} runtimeObject The object to render
+ * @param {gdjs.RuntimeScene} runtimeScene The gdjs.RuntimeScene in which the object is
+ */
+gdjs.VideoRuntimeObjectPixiRenderer = function(runtimeObject, runtimeScene) {
+  this._object = runtimeObject; // Keep a reference to the object to read from it later.
+
+  // Here we're going to create a video text as an example.
+  if (this._pixiObject === undefined) {
+    this._textureVideo = new PIXI.Texture.fromVideo(
+      "C:/Users/RTX-Bouh/Desktop/test_video_GD.mp4"
+    );
+
+    //Setup the PIXI object:
+    this._pixiObject = new PIXI.Sprite(this._textureVideo);
+
+    this._pixiObject._texture.baseTexture.source.pause();
+  }
+
+  //this._pixiObject.anchor.x = 0.5;
+  //this._pixiObject.anchor.y = 0.5;
+  runtimeScene
+    .getLayer("")
+    .getRenderer()
+    .addRendererObject(this._pixiObject, runtimeObject.getZOrder());
+
+  //this.updatePosition();
```

Comment:
I think it's a good idea to call updatePosition, and maybe updateAngle and updateOpacity so that you're 100% sure your renderer is synchronized with the object